caddy/Caddyfile

```diff
@@ -1,4 +1,4 @@
-:80
+:8080
```

Comment:
It's also necessary to fix the port number in the README file

Reply:
fixed